packages/algob/test/mocks/tx.ts

```diff
@@ -39,12 +39,77 @@ export const mockSuggestedParam: SuggestedParams = {
 	genesisHash: "SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=",
 };
 
+const account = algosdk.generateAccount();
+const addr = algosdk.decodeAddress(account.addr);
+
+export const TXN_OBJ = {
```

Comment:
maybe we can add a comment here about the type or just link do documentation

Reply:
Done